.devcontainer/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Externpro Centos 7 Development Container" \
+  name="externpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes --quiet update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
+    git \
+    gtk2-devel \
+    make \
+    mesa-libGL-devel \
+    mesa-libGLU-devel \
+    ncurses-devel \
+    ninja-build \
+    python-devel \
+    wget; \
+  yum --assumeyes --quiet clean all; \
+  ln -s cmake3 /usr/bin/cmake; \
```

Comment:
do you also want to add ccmake? `ln -s ccmake3 /usr/bin/ccmake; \`

Reply:
I added the symlink in #4.